.pre-commit-config.yaml

```
@@ -60,3 +60,32 @@ repos:
       entry: from \.[\.\w]* import
       types: [python]
       language: pygrep
+    - id: no-internal-links
+      name: Check no internal links are in the docs
+      description: >-
+        'A quick check for the links in the intersphinx '
+        'mapping inside `docs/source/`'
+        'See docs/source/conf.py for more information.'
+      files: ^docs/source/
+      # Files that should be ignored:
+      # - 404.md
```

```suggestion
      # - 404.md
      # - contributing_release_checklist.md
```

One of the steps in the release checklist is checking that the documentation has been built correctly and is served and available on readthedocs. It has the url to be checked written explicitly to make sure the task is clear even if docs or cross-referencing doesn't work as expected.